compiler/rustc_session/src/session.rs

```diff
@@ -858,12 +861,12 @@ impl Session {
         // If a target requires unwind tables, then they must be emitted.
         // Otherwise, we can defer to the `-C force-unwind-tables=<yes/no>`
         // value, if it is provided, or disable them, if not.
-        if self.panic_strategy() == PanicStrategy::Unwind {
-            true
-        } else if self.target.requires_uwtable {
+        if self.target.requires_uwtable {
```

Comment:
Preexisting, but this could be more conventionally be written as

```rust
self.target.requires_uwtable || self.opts.cg.force_unwind_tables.unwrap_or(...)
```